templates/_rows_and_columns.html

```diff
@@ -1,16 +1,40 @@
-<table>
+<table id="dbresults">
     <thead>
         <tr>
-            {% for column in display_columns %}<th scope="col">{{ column }}</th>{% endfor %}
+            <th scope="col">Link</th>
+            <th scope="col">First name</th>
+            <th scope="col">Middle initial</th>
+            <th scope="col">Last name</th>
+            <th scope="col">Organization</th>
+            <th scope="col">YTD gross earnings</th>
         </tr>
     </thead>
     <tbody>
     {% for row in display_rows %}
-        <tr>
+        <tr class="{{ loop.cycle('odd', 'even') }} fancyfloat">
```

Comment:
You can use [`:nth-child(even)`](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_boxes/Styling_tables) in CSS instead of adding an explicit class. We don't care about any browser too old to have nth-child.

Reply:
done